ets-demo/README.rst

```diff
@@ -8,6 +8,27 @@ packages can be used.
 
 The actual demonstration materials are not provided by this package.
 
+Installation
+------------
+
+The application requires a GUI backend to run, and a few options are supported.
+
+To install from PyPI with PySide2 dependencies::
+
+    $ pip install etsdemo[pyside2]
+
+Or with PyQt5 dependencies::
+
+    $ pip install etsdemo[pyqt5]
+
+Or with wxPython dependencies::
+
+    $ pip install etsdemo[wx]
```

Comment:
The installation instruction is included for wx for completeness. There are quite a few issues with wx to the extent that the application isn't that useable on wx (e.g. #1118). So the mention here could be a false advertisement. I could add a warning here to note that there are known issues with wx?

Reply:
A note about the wx issues sounds good to me.

Reply:
Done. Hope it looks alright.